Simulator/Recorders/XmlRecorder.h

```diff
@@ -67,49 +67,59 @@ class XmlRecorder : public Recorder {
    }
 
    // Getter method for neuronName_ (only included during unit tests)
```

Comment:
Let's use "accessor" instead of "getter".

Reply:
How about the getter method name? Should I also change the method name getHistory() to accessHistory()?